packages/core/contracts/financial-templates/perpetual-multiparty/PerpetualPositionManager.sol

```diff
@@ -280,8 +279,7 @@ contract PerpetualPositionManager is FundingRateApplier {
         PositionData storage positionData = _getPositionData(msg.sender);
         require(
```

Comment:
I feel that this one might get hit more frequently than some of the others.

Reply:
agreed ill re-add i have 500 bytes to work with